### PR TITLE
https cert configuration

### PR DESCRIPTION
**Pull request by mhenriks**
**What this PR does / why we need it**:

We added `storage.import.certConfigMap` for registry import, so why not use it for https import as well

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: cluster/clean.sh [hunk 1]
```diff
@@ -19,29 +19,26 @@ if [ -f "${OPERATOR_MANIFEST}" ]; then
     ./cluster/kubectl.sh delete --ignore-not-found -f "${OPERATOR_MANIFEST}"
 fi
 
-# Work around https://github.com/kubernetes/kubernetes/issues/33517
-./cluster/kubectl.sh delete ds -l 'operator.cdi.kubevirt.io' -l "cdi.kubevirt.io" -n ${NAMESPACE} --cascade=false --grace-period 0 2>/dev/null || :
-
-# Delete all traces of kubevirt
-namespaces=(default kube-system ${NAMESPACE})
+# Ecerything should be deleted by now, but just to be sure
```
Comment (awels) [inline cluster/clean.sh]: Everything

File: tests/utils/configmaps.go [hunk 2]
```diff
@@ -1,50 +1,66 @@
...
-func CopyConfigMap(client kubernetes.Interface, srcNamespace, srcName, destNamespace, destName string) error {
+func CopyConfigMap(client kubernetes.Interface, srcNamespace, srcName, destNamespace, destName string) (string, error) {
 	src, err := client.CoreV1().ConfigMaps(srcNamespace).Get(srcName, metav1.GetOptions{})
 	if err != nil {
-		return err
+		return "", err
+	}
+
+	if destName == "" {
+		destName = srcName + "-" + strings.ToLower(util.RandAlphaNum(8))
```
Comment (awels) [inline tests/utils/configmaps.go]: Maybe I am missing something, why are we creating a config map with a partially random name? Its getting copied to a new namespace which already has a randon name, so what is the reason to have the same thing for the config map?

Comment (mhenriks) [reply tests/utils/configmaps.go]: Not strictly necessary given that tests are running serially.  But I like the idea of each test having its own distinct data.  Which will be necessary if we ever run tests in parallel (which would be nice)

Comment (awels) [reply tests/utils/configmaps.go]: But each test creates its own namespace, which will be unique, even the ones in a 'table'

Comment (mhenriks) [reply tests/utils/configmaps.go]: Okay, I was confused.  I thought that a single namespace was created for all of the transport tests.  Did not see that the framework adds a before/after each calback that creates/deletes the namespace.

I still think this randomization is a nice to have.

Comment (awels) [reply tests/utils/configmaps.go]: It doesn't bother me that much, just wanted to point it out.

**Comment by mhenriks** [comment]
ci test please

**Pull request merged**